pages/portfolio.html

```diff
@@ -4,9 +4,76 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Amber's Portfolio</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
+  <link href="../styles/portfolio.css" rel="stylesheet"/>
+  <link href='https://fonts.googleapis.com/css?family=Amatic SC' rel='stylesheet'>
 </head>
+
+
 <body>
-  
-</body>
-</html>
\ No newline at end of file
+    <section>
+        <nav class="navigation">
+            <a href="index.html" >Home</a>
+            <a href="about.html" >About</a>
+            <a href="portfolio.html" >Portfolio</a>
+        </nav>
+    </section>
+
+
+    <div class="container">
+      <div class="wrapper">
+        <div class="item" style="background:rgb(200, 245, 244)">
+          <iframe src="https://drive.google.com/file/d/1hYnsGqC09_U5R_fQNZiexj4EekjGB1uk/preview" width="840" height="500" allow="autoplay" alt="A video showing a playthrough of the snowman game."></iframe>
+        </div>
+        <div class="item" stlye="background: rgb(200, 245, 244)">
+          <h1>Adagrams</h1>
+          <h3>This Project worked similarly to classic word games like scrabble, or bananagrams. First the code generates
+            10 random letters, each with different point values. Then it's up to the user to try to come up with a word
+            using available letters, to get the highest score possible! This project had a lot of focus on getting comfy
+            with dictionaries and iteration.
+          </h3>
```

Comment:
It looks like you used the `h1` and `h3` tags for styling.  Instead you should use CSS classes to change the size.

The `h1`, `h2`, etc tags are important for conveying information to things like screen readers and should be able to be put into a table of contents that makes sense.  (With a single `h1` at the top followed by `h2` on the lower level, `h3` on the level below that, etc.)